### PR TITLE
Task 3 Calculator

### PR DESCRIPTION
**Pull request by ViktoriiaKhatniuk**

### CODE REVIEW

**Review by Hmeliumm — CHANGES_REQUESTED** [review]
Out directory still there(

File: .gitignore [hunk 1]
```diff
@@ -15,5 +15,8 @@
 *.iml
 **/*.idea
 
+*/out
+*.class
```
Comment (Hmeliumm) [inline .gitignore]: You don't need this, if out/ folder will be ignored

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (ViktoriiaKhatniuk) [reply .gitignore]: I understand. I tried but  I can not remove out directory from git.

**Review by ViktoriiaKhatniuk — COMMENTED** [review]
ok

**Pull request merged**